setup.py

```diff
@@ -4,9 +4,15 @@
 setup(
     name="fms",
     version="0.0.1",
-    author="Brian Vaughan, Joshua Rosenkranz, Antoni Viros i Martin, Davis Wertheimer, Supriyo Chakraborty, Raghu Kiran Ganti, ...",
-    author_email="bvaughan@ibm.com",
+    author="Brian Vaughan, Joshua Rosenkranz, Antoni Viros i Martin, Davis Wertheimer, Supriyo Chakraborty, Raghu Kiran Ganti",
+    author_email="bvaughan@ibm.com, jmrosenk@us.ibm.com, aviros@ibm.com, Davis.Wertheimer@ibm.com, supriyo@us.ibm.com, rganti@us.ibm.com",
```

Comment:
I believe this will generate a broken `mailto:...` link on pypi.

We can probably add a better attribution list elsewhere but we should probably use a single email address here. A distribution list of fms devs would be ideal, do we have a team distribution list here?

Reply:
Yes, will keep this open til we have one and update it with the proper email.

Reply:
In the meantime I think we should put any valid email so that the `mailto:` link won't be broken on pypi